### PR TITLE
Feature header responsive fix

### PR DESCRIPTION
**Pull request by JohanLaubscher**
Added Jquery and updated the way the header is rendered depending on mobile devices

### CODE REVIEW

File: ChatterBox/try-app/src/container/app/index.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Header from '../Header'
 import Settings from '../profile-settings/profile-settings.js'
 import ChatForm from '../ChatForm/chatFrom.js'
 
+window.$ = window.jQuery = require('jquery')
```
Comment (KimberleighBeyl) [inline ChatterBox/try-app/src/container/app/index.js]: Don't use JQUERY!!!!!!!!

File: ChatterBox/try-app/src/container/Header/index.js [hunk 2]
```diff
@@ -4,38 +4,91 @@ import { TextField, RaisedButton } from 'material-ui'
...
-      <div class="dropdown">
-        <button class="dropdown-button btn header-button"><span class="header-button-label">NEW GROUP</span></button>
-        <div class="dropdown-content">
-          <TextField
-            hintText="Group Name"
-          />
+class Header extends React.Component {
+  constructor() {
+    super();
+    this.state = {
```
Comment (ReinhardtCromhout) [inline ChatterBox/try-app/src/container/Header/index.js]: If the windows size has changed it is potentially important for all components, not just the header. Therefore the window innerWidth needs to set in the redux store and not in the component state. You need to use Redux.

File: ChatterBox/try-app/src/container/app/index.js (hunk 1 above)
Comment (JohanLaubscher) [reply ChatterBox/try-app/src/container/app/index.js]: It's only for bootstrap.js to work, it's not used anywhere else

File: ChatterBox/try-app/src/container/Header/index.js (hunk 2 above)
Comment (JohanLaubscher) [reply ChatterBox/try-app/src/container/Header/index.js]: cool will take a look at it

File: ChatterBox/try-app/src/container/app/index.js (hunk 1 above)
Comment (KimberleighBeyl) [reply ChatterBox/try-app/src/container/app/index.js]: Does not make sense send me the link

Comment (JohanLaubscher) [reply ChatterBox/try-app/src/container/app/index.js]: I used a navbar of bootstrap which required the bootstrap.js file. Bootstrap.js is using jQuery in order to work so I included it. However, I forgot I removed that navbar so the jquery is not neccessary anymore. Will remove it.

Comment (KimberleighBeyl) [reply ChatterBox/try-app/src/container/app/index.js]: Thank you

File: ChatterBox/try-app/package.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
     "bootstrap": "^3.3.7",
     "css": "^2.2.1",
     "history": "^4.7.2",
+    "jquery": "^3.2.1",
```
Comment (KimberleighBeyl) [inline ChatterBox/try-app/package.json]: remove

**Comment by JohanLaubscher** [comment]
Merge screwed up this branch so this should not be merged to develop. It will overwrite the work. I will create a new branch and add my work again.

**Pull request closed without merging**